client/stylesheets/_toCat.scss

```diff
@@ -2513,7 +2516,8 @@ i:hover {
 }
 
 .clickDropper li ul li a {
-	padding: 15px 20px 15px 15px !important;
+	padding: 15px 0px 15px 15px !important;
```

Comment:
!important should not be used<br>Line should be indented with spaces, not tabs<br>Properties should be ordered font-size, padding<br>`0px` should be written without units as `0`